pkg/test/masteruserrecord/master_user_record.go

```diff
@@ -134,17 +127,16 @@ func TierName(tierName string) UaInMurModifier {
 	}
 }
 
-func Namespace(nsType, revision string) UaInMurModifier {
+func Namespace(oldTemplateRef, newTemplateRef string) UaInMurModifier {
```

Comment:
thanks for the review @MatousJobanek , i was wondering if it's okay to keep this the way it is as it is a bit easier for me to understand what's happening in the tests

Reply:
I believe that it would be OK - it also make sense using the function:
```go
murtest.Namespace("ide", "54321")`
```
in the sense that for `ide` namespace I want so use revision `54321`

To be honest, the current solution doesn't seem to be clear to me:
```go
murtest.Namespace("basic-dev-123abc", "admin-ide-54321")
```
if you want to keep this way of specifying/replacing the templateRef, then please rename the function as well:
```go
murtest.ReplaceTemplateRef("basic-dev-123abc", "admin-ide-54321")
```
or just 
```go
murtest.ReplaceRef("basic-dev-123abc", "admin-ide-54321")
```

anyway, the downside that I see is that you need to know the original value of the `TemplateRef` field

Reply:
I'm not really happy with this change - what if the `oldTemplateRef` is not found? then the tests wouldn't verify the intended functionality. It looks like error-prone to me.

Reply:
@MatousJobanek hmm yeah, looks like it could be prone to error - how would you suggest changing this function?

Reply:
@MatousJobanek i guess as you first suggested - maybe what we could do is have the func take in `revision` and `nsType` and then we can find the `templateRef` which contains the nsType passed in and just reset the `revision` - does this sound good?

Reply:
yes, that sounds good :+1:

Reply:
fixed here: 3440b7317af0545532cb6fc736bc6e569f933c85